client-cli/src/command/wallet_command.rs

```diff
@@ -25,6 +25,49 @@ impl WalletCommand {
         }
     }
 
+    fn get_mnemonics<T: WalletClient>(wallet_client: &T) -> String {
+        let mut mnemonics: String;
+        loop {
+            println!("== hd wallet setup==");
+            println!("1. create new mnemonics");
+            println!("2. restore from mnemonics");
+            println!("enter command=");
+
+            let a = quest::text().unwrap();
+            if a == "1" {
+                mnemonics = wallet_client.new_mnemonics().unwrap();
+            } else if a == "2" {
+                println!("enter mnemonics=");
+                mnemonics = quest::text().unwrap().to_string();
```

Comment:
Instead of accepting mnemonics as String, should we accept it as SecStr?

Reply:
But this would cause the confirmation of mnemonics below not possible

Reply:
i'll check to use SecStr

Reply:
done